templates/nginx.conf

```diff
@@ -27,7 +27,7 @@ server {
     ssl_ciphers 'ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256';
     ssl_session_timeout  10m;
     ssl_session_cache shared:SSL:10m;
-    ssl_session_tickets off;
+    ssl_session_tickets on;
```

Comment:
I was getting an SSL error very strangely on my test host, and this was the fix.